reference/conanfile/tools/intel_cc.rst

```diff
@@ -0,0 +1,247 @@
+.. _conan_tools_intel_cc:
+
+
+IntelCC
+=========
+
+.. warning::
+    These tools are **experimental** and subject to breaking changes.
+
+
+Available since: `1.41.0 <https://github.com/conan-io/conan/releases>`_
+
+
+This section explains how to use the new Intel oneAPI `DPC++/C++ <https://software.intel.com/content/www/us/en/develop/documentation/oneapi-dpcpp-cpp-compiler-dev-guide-and-reference/top.html>`_ and
+`Classic <https://software.intel.com/content/www/us/en/develop/documentation/cpp-compiler-developer-guide-and-reference/top.html>`_ compilers in Conan.
+
+.. note::
+    This new compiler is developed as a new component in ``conan.tools.intel`` and it will be automatically applied through ``CMakeToolChain`` or ``MSBuildToolchain`` generators.
```

Comment:
I (a frequent Conan user) don't know what does it mean and why I need to know it.